hvpy/api_groups/jpeg2000/get_jp2_image.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+from datetime import datetime
+
+from pydantic import validator
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+class getJP2ImageInputParameters(HvpyParameters):
+    date: datetime
+    sourceId: int
+    jpip: Optional[bool] = False
+    Json: Optional[bool] = False
+
+    # Use validator to convert date to isoformat with suffix 'Z'
+    @validator("date")
+    def convert_date_to_isoformat(cls, v):
+        return v.isoformat() + "Z"
+
+    def get_output_type(self):
+        if self.Json == False and self.jpip == False:
+            return OutputType.Raw
+        elif self.Json == True and self.jpip == True:
+            return OutputType.Json
+        elif self.Json == False and self.jpip == True:
+            return OutputType.String
```

Comment:
You're missing the case where `self.Json == True and self.jpip = False` which should return `OutputType.Raw`

You could just add a catch-all `return OutputType.Raw` at the bottom.